tests/network/vmi_networking.go

```diff
@@ -882,14 +882,7 @@ var _ = SIGDescribe("[Serial][rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com]
 				Expect(err).ToNot(HaveOccurred())
 				Expect(vmi.Status.Phase).To(Equal(v1.Running))
 
-				Eventually(func() error {
-					err := ping(podIP)
-					if err != nil {
-						return err
-					}
-
-					return nil
-				}, 120*time.Second).Should(Succeed())
+				Expect(ping(podIP)).To(Succeed())
```

Comment:
Is it possible to check the mac address of the bridge on the virt-launcher pod?
If will be ugly (because we do not assert the API, but some inner detail), but depending on this ping timing is unsafe.

Reply:
I don't follow the meaning of unsafe here; the point of preserving the MAC between migrations is to remove the need for the ARP tables to stabilize, which should preserve connectivity from the outside world to the guest, once the VM is considered `running`.

Is there something I am missing here ? 

As a side note, yes, it is possible to check the mac of the bridge. And yes, it is ugly. And it doesn't prove anything about preserving the connectivity of the VM once migrated, which is this PR's objective.

Reply:
FWIW, I'm running this in a loop w/ `untilItFails` flag. So far, I'm running iteration #20:

```bash
$ FUNC_TEST_ARGS='-untilItFails -race' make functest
...
Ran 2 of 813 Specs in 197.550 seconds
SUCCESS! -- 2 Passed | 0 Failed | 0 Pending | 811 Skipped
PASS | FOCUSED

All tests passed...
Will keep running them until they fail.
This was attempt #19
Maybe you should stop now?
...
```

Reply:
> Is there something I am missing here ?

The ping may fail for other reasons, like guest CPU right after the migration or MAC address duplication on the switch (the mac appears on a different port, so it will not reach to the right destination).
I do not think it is safe to assume this is the only reason to have echo packets lost.

> As a side note, yes, it is possible to check the mac of the bridge. And yes, it is ugly. And it doesn't prove anything about preserving the connectivity of the VM once migrated, which is this PR's objective.
> 
> EDIT: above I mean that "preserving the connectivity of the VM once migrated" is the issue this PR is trying to fix.

I would use "improve" not "solve" here. I am just pointing out that there may be other factors that affect a smooth migration and connectivity.

There is also a logical problem with what you are saying: If the mac is equivalent, we can be sure the ping will work fine (lets assume this is true). However, you cannot assume the other direction, right? So what protects us from someone breaking this feature eventually?

Reply:
I am just worried that we are *not* testing this behavior (that the mac is explicit set to a hard-coded address and is expected to be the same on all similar virt-launchers).

I guess my logical claim is this: If this will get broken and the mac address will no longer be the same as we expect, then the ping may do pass from time to time, becoming flaky and letting us wander what happened.

If you are not worry about that, then lets proceed, not covering this explicitly is not worse than what we have today.
If we will encounter this in the future, we can handle it then.